src/db/asset_general.cc

```diff
@@ -266,26 +266,28 @@ db_reply_t
      zhash_t         *extattributesRO)
 {
     LOG_START;
+    const char* msg1 = TRANSLATE_ME ("Element '%s' cannot be processed because of conflict. Most likely duplicate entry.");
     if (DBAssets::extname_to_asset_id(element_name) != -1) {
         db_reply_t ret;
         ret.status     = 0;
         ret.errtype    = DB_ERR;
         ret.errsubtype = DB_ERROR_BADINPUT;
         ret.rowid      = 8;
-        ret.msg        = std::string ("Element '").append (element_name).append ("' cannot be processed because of conflict. Most likely duplicate entry.");
+        ret.msg        = std::string (zsys_sprintf (msg1, element_name));
```

Comment:
This approach wouldn't work as intended - since TRANSLATE_ME contains only string, not it's argument, it would be impossible to use this macro to encapsulate to_be_translated string into JSON and append it's variables into it so UI can translate it properly afterwards.